### DISCUSSION TITLE
How to Install the Pikafish Engine in a React Native App

### DISCUSSION THREAD

**Discussion by zhongzhubailong** [official-pikafish/Pikafish#69] [General]
Hi, we hope to install the Pikafish local engine in our React Native App. We would like to know if this can be achieved using pure JavaScript or an npm package, or if the engine can be compiled into Node.js and integrated with React Native. We would be very grateful!
你好，我们希望在我们的React Native App中安装Pikafish本地引擎。我们想知道是否可以使用纯JavaScript或npm套件实现这一目标，或者是否可以将引擎编译成Node.js并将其与React Native集成。万分感激！

**Comment by Asad520** [comment]
@zhongzhubailong Were you able to accomplish this? I want to do the same for my react native app
